lib/utils/get-registry-ky.ts

```diff
@@ -1,29 +1,47 @@
-import ky from "ky"
+import ky from "ky";
 
-// Flag to identify if we're running inside runframe specifically
+// Flag to identify environment configuration
 declare global {
   interface Window {
-    __RUNFRAME_REGISTRY_BASE_URL__?: boolean
+    __RUNFRAME_REGISTRY_BASE_URL__?: boolean;
+    REGISTRY_API_BASE_URL?: string; // For custom API URL
   }
 }
 
 // Create a function that provides the configured ky instance
-// This ensures we check the flag every time we need the client
+// This ensures we check the flag/config every time we need the client
 export function getRegistryKy() {
-  // Check flag at execution time rather than module load time
-  const useRegistryPrefix =
-    typeof window !== "undefined" &&
-    window.__RUNFRAME_REGISTRY_BASE_URL__ === true
-
-  // Set the appropriate prefix based on the environment
-  const registryApiBaseUrl = useRegistryPrefix
-    ? "/registry"
-    : "https://registry-api.tscircuit.com"
+  // For runframe specific handling
+  const isRunframe = 
+    typeof window !== "undefined" && 
+    window.__RUNFRAME_REGISTRY_BASE_URL__ === true;
```

Comment:
ONLY use REGISTRY_API_BASE_URL. It is set to `/registry` in RunFrame and `/api` in tscircuit.com during local development

```suggestion
```